GVFS/GVFS.Common/AzDevOpsOrgFromNuGetFeed.cs

```diff
@@ -0,0 +1,51 @@
+using System.Text.RegularExpressions;
+
+namespace GVFS.Common
+{
+    public class AzDevOpsOrgFromNuGetFeed
+    {
+        /// <summary>
+        /// Given a URL for a NuGet feed hosted on Azure DevOps,
+        /// return the organization that hosts the feed.
+        /// </summary>
+        public static bool TryParseOrg(string packageFeedUrl, out string orgName)

```

Comment:
Interesting point. I would rather not add the responsibility of parsing these strings to the `NuGetUpgraderConfig` class.

I can:
1) Make these instance methods on the `AzDevOpsOrgFromNuGetFeed` class (until I find a better name for this class). This would address the question about calling the regex multiple times.

2) Bring these back to static methods on the `NuGetUpgrader` class, or even work it to make it only called 1 time (possible as a property of NuGetUpgrader...)

However, This is not being called in a perf critical code currently, and I don't think this is likely to be called multiple times in a perf critical section...